src/components/WebexInterstitialMeeting/WebexInterstitialMeeting.scss

```diff
@@ -1,14 +1,20 @@
 .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-interstitial-meeting {
   position: relative;
   display: flex;
+  align-self: stretch;
```

Comment:
these should come first (relation to the parent)

Reply:
actually, these should come from the parent, the component should not set its own width/height